### PR TITLE
restructure

### PR DESCRIPTION
**Pull request by KINGSABRI**
Please check this structure. if you accept it, we can build on it

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="Rubocop" enabled="false" level="WARNING" enabled_by_default="false" />
+  </profile>
+</component>
```
Comment (sdaubert) [inline .idea/inspectionProfiles/Project_Default.xml]: Could you remove your editor's project?

File: Supported-Protocols.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+|    DNS   	|     ✔     	|
+|   Dot11  	|     ✔     	|
+|   Dot1q  	|     ✔     	|
+|    EAP   	|     ❗      	|
+|    ESP   	|     ✔     	|
+|    GRE   	|     ✔     	|
+|    GRE   	|     ✔     	|
+|    LLC   	|     ✔     	|
+|    IKE   	|     ✔     	|
+|   DHCP   	|     ❌ 	    |
```
Comment (sdaubert) [inline Supported-Protocols.md]: DHCP is not (yet?) supported 😃

**Comment by sdaubert** [comment]
Two minors changes please, and that will be OK.

**Comment by KINGSABRI** [comment]
wat's your observation? I would can fix it

File: Supported-Protocols.md (hunk 2 above)
Comment (KINGSABRI) [reply Supported-Protocols.md]: :dancer: I was hoping to be supported one day :D

**Pull request merged**